bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
 
   puts "\n== Resetting tests =="
   system! "bin/rails test:vmdb:setup"
```

Comment:
@jrafanie Comments like these are how I can spot when someone didn't read the PR description:

> The exception currently the `test:vmdb:setup` task, which errors out when a conflicting Rails environment is already loaded. To avoid a complex solution for this, it has been left as a `system!` call for now.

So yes, that is indeed the reason.